index.js

```diff
@@ -243,6 +243,15 @@ exports.stringify = (object, options) => {
 	}, options);
 
 	const formatter = encoderForArrayFormat(options);
+
+	if (options.skipNullAndUndefined) {
+		for (const key of Object.keys(object)) {
+			if (object[key] === undefined || object[key] === null) {
+				delete object[key];
```

Comment:
Here you're mutating the passed object.

Reply:
I suppose `stringify` function has to be pure.

Reply:
Makes sense. Will update it now.

Reply:
Fixed in #d0b21db.